### DISCUSSION TITLE
wslview doubt

### DISCUSSION THREAD

**Discussion by SufyaanKhateeb** [wslutilities/wslu#219] [Q&A]
Why does wslview not work outside the mnt disks?

If anyone knows how to use wslview outside the mnt disks then please help.

My use case eg:
I want to open google, so I want to run the command like (wslview "https://www.google.com/") outside the mnt disks.

Thanks.

**Comment by car-meows** [comment]
This is a known bug, and I am still doing tests on it to make sure it works properly.
